google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardDeployTask.java

```diff
@@ -63,6 +65,16 @@ public void execute(ProcessStartListener startListener) {
         .withLabel(isFlexCompat ? "flex-compat" : "standard")
         .ping();
 
+    if (CloudSdkService.getInstance().validateCloudSdk().contains(
+        CloudSdkValidationResult.CLOUD_SDK_NOT_FOUND)) {
+      deploy.getCallback().errorOccurred(
+          GctBundle.message("appengine.cloudsdk.location.invalid.message") + " "
+              + CloudSdkService.getInstance().getSdkHomePath());
+      logger.warn(GctBundle.message("appengine.cloudsdk.location.invalid.message") + " "
```

Comment:
do we need to log this? Also log messages should not come from the resource bundle.